cabal.project

```diff
@@ -41,3 +41,14 @@ write-ghc-environment-files: always
 -- IMPORTANT
 -- Do NOT add more source-repository-package stanzas here unless they are strictly
 -- temporary! Please read the section in CONTRIBUTING about updating dependencies.
+
+-- Add upper bound to io-classes-mtl:
+
+constraints:
+  io-classes-mtl < 0.1.2.0
+
+source-repository-package
+    type: git
+    location: https://github.com/newhoggy/polysemy-conc
```

Comment:
We don't use it directly.  The SRP is because `polysemy-conc` as a dependency that doesn't compile on Windows.

See: https://github.com/tek/polysemy-conc/pull/4